node-sdk-sample/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "sdk-sample",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "commonjs",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint src",
+    "build": "tsc",
+    "watch": "tsc -w"
```

should we add a script to run the sample?

```suggestion
    "watch": "tsc -w",
    "start": "npm run build && node lib/script.js"
```